common/app/views/fragments/atoms/snippets/guide.scala.html

```diff
@@ -32,5 +32,16 @@
       @Html(item.body)
     </div>
   }
+} {
+  @for(
+    img <- guide.image;
+    asset <- img.allImages.headOption;
+    credit <- asset.credit
+  ) {
+    <div class="explainer-snippet__credit">
+      @fragments.inlineSvg("information", "icon") 
+      Photograph: @credit
```

Comment:
The `credit` field will include the "Photograph:" bit, see https://github.com/guardian/atom-workshop/pull/175

Reply:
Also, might `@Html(credit)` be necessary?